components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -780,32 +780,30 @@ void RewardsServiceImpl::TriggerOnWalletInitialized(int error_code) {
 void RewardsServiceImpl::TriggerOnWalletProperties(int error_code,
     std::unique_ptr<ledger::WalletInfo> wallet_info) {
   std::unique_ptr<brave_rewards::WalletProperties> wallet_properties;
+  for (auto& observer : observers_) {
+    if (wallet_info) {
```

Comment:
This check needs to be added to this line as well https://github.com/brave/brave-core/pull/576/files#diff-a5d16be9c9b52a6df7b866b1d75d3f5fR810

Reply:
I pushed it inside the loop. Should it be added to the `if` line?

Reply:
I think this shouldn't be inside observer for loop, because that is for extension. Should be separate, but have if check for wallet_info

Reply:
Done